devfiles/che7-preview-plugin-dev.yaml

```diff
@@ -0,0 +1,26 @@
+---
+displayName: Che 7 Dev
+description: Workspace.next sidecars and Theia as IDE with the tooling for plugin development
```

Comment:
This stack looks like sample, as far as I remember it was used on EclipseCon 2018.
@l0rd Do you have an idea whether we still need since we have another stack for Theia Development contributed and used by IDE2 team https://github.com/eclipse/che-devfile-registry/pull/3/files#diff-c73cf9a15f50d1a8ca46c423fe23abf3R3 ?

Reply:
Please refer to https://github.com/eclipse/che/issues/12620. We need only the stacks listed in that issue.

Reply:
Thanks for sharing. Have not noticed this issue before

Reply:
Checked https://github.com/eclipse/che/issues/12620 and only these stacks are presented there
- apache-camel-springboot-che7.yaml
- dotnet.yaml
- java-gradle.yaml
- go.yaml
- java-maven.yaml
- python.yaml

Also there is mentioned Che-Theia Developer stack, but referenced is sunix's Devfile but not Che 7 stack that we have. So, the following stacks will be dropped out.
- che7-development-che-theia-and-theia-plugins.yaml
- che7-preview-plugin-dev.yaml
- java-mysql-theia-kubernetes.yaml
- che7-preview.yaml